PetaPoco.Tests.Integration/PetaPoco.Tests.Integration.csproj

```diff
@@ -53,6 +53,9 @@
     <Compile Remove="Databases\MSAccessTests\**" />
     <Compile Remove="Providers\MSAccessTestProvider.cs" />
     <EmbeddedResource Remove="Scripts\MSAccessBuildDatabase.sql" />
+    <PackageReference Include="Oracle.ManagedDataAccess.Core">
+      <Version>3.21.120</Version>
+    </PackageReference>
```

Comment:
Given the current structure of this project file, this probably should be dropped in with the other package references IMO. It seems there's no right way of laying out these files, but grouping this with the list removal items is a sure-fire way of confusing yours truly some late night eventually. 😆

Reply:
All set.